### DISCUSSION TITLE
Please add the out painting tool

### DISCUSSION THREAD

**Discussion by SunGreen777** [lllyasviel/stable-diffusion-webui-forge#2060] [General]
Please add the out painting tool to the Space tab so that it works with flux.

**Comment by JohnneyTexas** [comment]
I also beg you to do it !
I wish powerful outpainting function like Fooocus was included in Forge !

**Reply by SunGreen777** [reply]
> I also beg you to do it ! I wish powerful outpainting function like Fooocus was included in Forge !
Yes, that’s exactly what I’m writing about, thank you for sharing my interest.
